src/utils/helpers/formatNumbersToString.ts

```diff
@@ -21,3 +25,8 @@ export const formatBytesString = (bytes: number): string => {
 
     return `${bytes.toFixed(decimalPlaces)} ${units[i]}`;
 };
+
+export function unixTimeToDate(unixTimeMs: number) {
```

Comment:
Good call moving helpers to this file! 
Can we add typing for this function? 
Also, since it is returning a string and not a date object, should we stick to the previous name?